pkg/innerring/innerring.go

```diff
@@ -390,12 +390,34 @@ func New(ctx context.Context, log *zap.Logger, cfg *viper.Viper, errChan chan<-
 		}
 	}
 
+	isLocalConsensus := isLocalConsensusMode(cfg)
```

Comment:
previous position after `err` check was better

Reply:
wanted to reduce the number of empty lines, returned